source/containerimage/source.go

```diff
@@ -55,7 +56,8 @@ type SourceOpt struct {
 
 type Source struct {
 	SourceOpt
-	g flightcontrol.Group[*resolveImageResult]
+	g  flightcontrol.Group[*resolveImageResult]
+	g2 flightcontrol.Group[*sourceresolver.AttestationChain]
```

Comment:
nit: maybe more descriptive?

```suggestion
	fgImageRes    flightcontrol.Group[*resolveImageResult]
	fgAttestChain flightcontrol.Group[*sourceresolver.AttestationChain]
```